docs/templates/hardware-matrix.rst.j2

```diff
@@ -30,10 +30,10 @@ supported by plugins bundled with tmt.
    * - :ref:`{{ requirement }}{# djlint:off #}</spec/hardware/{# djlint:on H025 #}{{ requirement }}>`
     {% for plugin, (enabled, note_id) in plugins.items() %}
         {% if enabled %}
-     - :supported:`yes`{% if note_id %} [{{ note_id }}]_{% endif %}
+     - ✅{% if note_id %} [{{ note_id }}]_{% endif %}
```

Comment:
Maybe should be centered as well, not sure how to do that with `list-table`

Reply:
no idea, I would not care about that now personally,